app/javascript/packages/address-search/components/full-address-search.tsx

```diff
@@ -28,8 +30,12 @@ function FullAddressSearch({
           {t('idv.failure.exceptions.post_office_search_error')}
         </Alert>
       )}
-      <PageHeading>{t('in_person_proofing.headings.po_search.location')}</PageHeading>
-      <p>{t('in_person_proofing.body.location.po_search.po_search_about')}</p>
+      {handleLocationSelect && (
+        <>
+          <PageHeading>{t('in_person_proofing.headings.po_search.location')}</PageHeading>
+          <p>{t('in_person_proofing.body.location.po_search.po_search_about')}</p>
+        </>
+      )}
```

Comment:
I added logic to display/not display PageHeader and the PO Search About text.  This logic is needed because we do not want to display both when this component is used in the Help Center, see the stuff in red.  handleLoctionSelect is being used to display/not display the Select buttons on the PO list.  It is to prevent the following when being used in the Help Center

![Screenshot 2023-09-27 at 1 09 36 PM](https://github.com/18F/identity-idp/assets/125507397/9ef44545-8b92-4158-a2f0-beb7d12f4814)